test/parallel/test-fs-stream-construct-compat-error-write.js

```diff
@@ -31,7 +31,7 @@ tmpdir.refresh();
     });
   });
 
-  fs.open(`${tmpdir.path}/dummy`, 'wx+', common.mustCall((err) => {
+  fs.open(`${tmpdir.path}/dummy`, 'wx+', common.mustCall((err, fd) => {
     debuglog('fs open() callback');
     assert.ifError(err);
```

Comment:
I think you want the close here, before creating the write stream

Reply:
Ah, now I understand what the test is actually trying to test. :-D

FWIW, putting it where I put it causes the test to pass, so it's definitely that open fd that's keeping the event loop open.